app/controllers/users_controller.rb

```diff
@@ -91,11 +91,86 @@ def set_vars_from_curr_user
 		@cuisine = current_user.cuisine
 	end
 
-
+	@all_recipes = {'items' => 
```

Comment:
We should remember to remove @all_recipes if necessary as refactoring in the next iteration.